packages/sfui/frameworks/react/vite.config.ts

```diff
@@ -21,9 +21,7 @@ export default defineConfig({
         'tailwind-config': path.resolve(__dirname, 'tailwind-config.ts'),
       },
       name: 'storefront-ui-react',
-      fileName: (format, entryName) => format === 'es'
-        ? `${entryName}.mjs`
-        : `${entryName}.${format}.js`,
```

Comment:
Nitpick: not needed, no?

Reply:
you mean formatting right? if so I removed it :)